core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1811,6 +1811,11 @@ public View getView(String name) {
         return viewGroupMixIn.getViews();
     }
 
+    @Exported     //Just to be exported - @Exported on default methods in interface is not supported
```

Comment:
It's the open question. Probably it would be best to have `allViews` exported instead of just the 1st level views the same as (effectively) all jobs are exported (being top level jobs). Nevertheless for backward compatibility `views` probably should be exported anyway.

@daniel-beck What visibility should be used to have it accessible via remote API, but on the other hand suppressed when root Jenkins elements are listed (unfortunately it's the first level element)?